tests/integration/modules/test_state.py

```diff
@@ -66,33 +66,84 @@ def test_show_highstate(self):
         """
         state.show_highstate
         """
-        high = self.run_function("state.show_highstate")
-        destpath = os.path.join(RUNTIME_VARS.TMP, "testfile")
-        self.assertTrue(isinstance(high, dict))
-        self.assertTrue(destpath in high)
-        self.assertEqual(high[destpath]["__env__"], "base")
+        top_sls = """
+        base:
+          '*':
+            - core
+            """
+
+        core_state = """
+        {{ salt['runtests_helpers.get_salt_temp_dir_for_path']('testfile') }}:
```

Comment:
We shouldn't need the helper to know this path now, it's `RUNTIME_VARS.TMP` ....

Reply:
I had thought about that too but since we're testing a state file run would we want to also make sure state files with jinja are working as expected as well?

Reply:
I'd create specific tests for that, if don't already have, which I doubt.

And that's just a custom helper module so we could pass those values at runtime....